agent/qrexec-agent.h

```diff
@@ -28,7 +28,7 @@
 
 int handle_handshake(libvchan_t *ctrl);
 void handle_vchan_error(const char *op);
-_Noreturn void do_exec(char *cmd);
+_Noreturn void do_exec(char *cmd, char *const *argument);
```

Comment:
Why? If preparation for replacement of `qubes-rpc-multiplexer`, better introduce this (and other places where you add `argument` parameter) in separate commit, as this is unrelated to socket-based services.

Reply:
Agreed.